src/main/kotlin/org/rust/ide/annotator/RsHighlightingMutableAnnotator.kt

```diff
@@ -81,7 +80,7 @@ class RsHighlightingMutableAnnotator : Annotator {
 
 val RsElement.isMut: Boolean
     get() = when (this) {
-        is RsPatBinding -> mutability.isMut || kind.isRefMut || type.let { it is TyReference && it.mutability.isMut }
+        is RsPatBinding -> mutability.isMut || type.let { it is TyReference && it.mutability.isMut }
```

Comment:
Maybe `type is TyReference && type.nmutability.isMut` is better? Or `type` is `var`?

Reply:
@fedochet `type` is a property with custom getter, so it's impossible.